### PR TITLE
added explanation and examples for imagepolicy admission plugin

### PR DESCRIPTION
**Pull request by gaurav-nelson**
For: https://bugzilla.redhat.com/show_bug.cgi?id=1565247

### CODE REVIEW

File: admin_guide/image_policy.adoc [hunk 1]
```diff
@@ -108,12 +108,48 @@ The ImageStream "bar" is invalid:
 
 [[image-policy-configuring-the-image-policy-admission-plug-in]]
 == Configuring the ImagePolicy Admission Plug-in
+To configure which images can run on your cluster, configure the ImagePolicy
+Admission plug-in in the `*_master-config.yaml_*` file. You can set one or more
+rules as required. Following are examples of some of the rules you can set:
```
Comment (bfallonf) [inline admin_guide/image_policy.adoc]: I'd say "Following are examples of some of the rules you can set" is not needed #minimalism

File: admin_guide/image_policy.adoc [hunk 2]
```diff
@@ -108,12 +108,48 @@ The ImageStream "bar" is invalid:
...
+  onResources:
+  - resource: pods
+  - resource: builds
+  reject: true
+  matchImageAnnotations:
+  - key: images.openshift.io/deny-execution <1>
+    value: "true"
+  skipOnResolutionFailure: true
+----
+<1> This annotation may be set by infrastructure that wishes to flag particular images as dangerous.
```
Comment (bfallonf) [inline admin_guide/image_policy.adoc]: I think the footnote above would serve better in the introduction on line 117? So:

"If a particular annotation has been deemed harmful, you can set a rule to reject all images that use a specific annotation. The following rejects all images using the `images.openshift.io/deny-execution` annotation"

Or something like that?

File: admin_guide/image_policy.adoc [hunk 3]
```diff
@@ -108,12 +108,48 @@ The ImageStream "bar" is invalid:
...
 
-.Annotated Example File
-====
+* *Enable user to run images from Docker Hub*:
++
+Use this rule to allow users to use images from docker hub:
```
Comment (bfallonf) [inline admin_guide/image_policy.adoc]: s/docker/Docker

I think?

Comment (bfallonf) [reply admin_guide/image_policy.adoc]: s/hub/Hub

as well?

File: admin_guide/image_policy.adoc [hunk 4]
```diff
@@ -108,12 +108,48 @@ The ImageStream "bar" is invalid:
...
+- name: allow-images-from-dockerhub
+  onResources:
+    - resource: pods
+    - resource: builds
+    matchRegistries:
+    - docker.io
+----
 
+Following is an example configuration for setting multiple ImagePolicy
+addmission plugin rules in the `*_master-config.yaml_*` file:
```
Comment (bfallonf) [inline admin_guide/image_policy.adoc]: s/plugin/plug-in

But also, does this need a heading like the other two examples above?

`
*Setting multiple ImagePolicy plug-ins*
`

**Comment by bfallonf** [comment]
@gaurav-nelson I made some nits and suggestions. Let me know if you have thoughts/disagree.

File: admin_guide/image_policy.adoc (hunk 4 above)
Comment (gaurav-nelson) [reply admin_guide/image_policy.adoc]: The previous two are example uses of using a single ImagePolicy plugin, this one is an example about how you can define multiple ImagePolicy plugins in `master-config` file. I will keep this separate,.

File: admin_guide/image_policy.adoc (hunk 2 above)
Comment (gaurav-nelson) [reply admin_guide/image_policy.adoc]: `If a particular annotation has been deemed harmful,` I mean to say: `If a particular image has been deemed harmful administrators can set this annotation to flag those images.`

**Comment by bfallonf** [comment]
@gaurav-nelson Sure. LGTM :cactus:

**Pull request merged**